datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -351,16 +351,17 @@ def _normalize_tags_type(self, tags, device_name=None, metric_name=None):
 
         if tags is not None:
             for tag in tags:
-                if not isinstance(tag, str):
-                    try:
-                        tag = tag.decode('utf-8')
-                    except UnicodeError:
-                        self.log.warning(
-                            'Error decoding tag `{}` as utf-8 for metric `{}`, ignoring tag'.format(tag, metric_name)
-                        )
-                        continue
-
-                normalized_tags.append(tag)
+                if not tag:
```

Comment:
since we expect `None` tags, let's explicitly do `if tag is None: continue` and leave the rest of the code unchanged

Reply:
✅